### PR TITLE
Consumer, generator and transformer template for all supported languages

### LINKED ISSUE #3: Add default scripts in package.json
For easier use of sequences it would be nice to add some starter scripts for user in templates, example for ts:
```
"scripts": {
    "build": "tsc -p tsconfig.json",
    "postbuild": "cp package.json dist/ && (cd dist && npm i --omit=dev)",
    "clean": "rm -rf ./dist ./*.tar.gz",
  },
```

### PR DESCRIPTION
**Pull request by gzukowski**

### CODE REVIEW

File: templates/ts-consumer/package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@scramjet/ts-simple",
+  "name": "@scramjet/ts-consumer",
   "version": "1.0.0",
   "description": "Scramjet Cloud Platform Sequence Typescript Template",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "predeploy" : "npm i && tsc -p tsconfig.json && cp -r package.json dist/ && mv node_modules dist"
```
Comment (ErykSol) [inline templates/ts-consumer/package.json]: this command will install all dependencies along with devDependencies, and move them to sequence package. Only dependencies should be added to sequence package. Proper command would be `tsc -p tsconfig.json && cp package.json dist/ && (cd dist && npm i --omit=dev)`

File: templates/ts-generator/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@scramjet/ts-generator",
+  "version": "1.0.0",
+  "description": "Scramjet Cloud Platform Sequence Typescript Template",
+  "main": "index.js",
+  "scripts": {
+    "predeploy" : "npm i && tsc -p tsconfig.json && cp -r package.json dist/ && mv node_modules dist"
```
Comment (ErykSol) [inline templates/ts-generator/package.json]: same case as in ts-consumer

File: templates/ts-transformer/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@scramjet/ts-transformer",
+  "version": "1.0.0",
+  "description": "Scramjet Cloud Platform Sequence Typescript Template",
+  "main": "index.js",
+  "scripts": {
+    "predeploy" : "npm i && tsc -p tsconfig.json && cp -r package.json dist/ && mv node_modules dist"
```
Comment (ErykSol) [inline templates/ts-transformer/package.json]: same case as in ts-consumer

**Review by ErykSol — CHANGES_REQUESTED** [review]
Fix packing redundant devDependencies in ts samples

**Comment by ErykSol** [comment]
Remember to change:
https://github.com/scramjetorg/create-sequence/blob/da9c3735359946747f21f61db65b92a12288104c/create-sequence.js#L11
Otherwise scripts will stop working after changes.

File: README.md [hunk 4]
```diff
@@ -16,3 +16,19 @@ npm init scramjetorg/sequence
 ```
 
 the main file and package.json file will be created out of `templates/default`.
```
Comment (ErykSol) [inline README.md]: templates/default no longer exist

File: README.md [hunk 5]
```diff
@@ -4,10 +4,10 @@ The [`Scramjet Cloud Platform`](https://docs.scramjet.org/platform) sequence pac
 initialization template ran by `npm init scramjetorg/sequence <template-name>`, for example:
 
 ```bash
-npm init scramjetorg/sequence py-simple
+npm init scramjetorg/sequence py-generator
 ```
 
-The command above will not create any sequence directory, main file and package.json will be created in a place from which `npm init ...` command was executed. We advise to create sequence directory first and then execute `npm init scramjetorg/sequence <template-name>` from there.
+The command above will not create any sequence directory, main file and package.json will be created in a place from which `npm init ...` command was executed. We advise to create sequence directory first and then execute `npm init scramjetorg/sequence <template-name>` from there. Then, if using typescript, use the `npm install` command.
```
Comment (ErykSol) [inline README.md]: typo: main file->files (for TS and PY there is more)

File: README.md [hunk 6]
```diff
@@ -16,3 +16,19 @@ npm init scramjetorg/sequence
 ```
 
 the main file and package.json file will be created out of `templates/default`.
+
+## Running
+Open a terminal in the folder of the selected template, then use:
+
+```bash
+# If you run this sample on Self Hosted Hub, please start it with process adapter option:
+DEVELOPMENT=1 sth --runtime-adapter=process
```
Comment (ErykSol) [inline README.md]: not sure if DEVELOPMENT flag is required here.... please verify it not checked already

**Review by ErykSol — APPROVED** [review]
Looks good, small typos to fix, but scirpts looks fine :)

**Review by a-tylenda — APPROVED** [review]

**Pull request merged**